app/build.gradle

```diff
@@ -22,10 +22,10 @@ android {
     }
 }
 
-ktlint {
-    android = true
-    reporters = [ReporterType.PLAIN, ReporterType.CHECKSTYLE]
-}
+//ktlint {
+//    android = true
+//    reporters = [ReporterType.PLAIN, ReporterType.CHECKSTYLE]
+//}
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/errysuprayogi/ReviewMe/blob/2853431aaa2d2b491e997e69b726255630c3820f/app/build.gradle#L31"></span>A newer version of org.jetbrains.kotlin:kotlin-stdlib-jdk7 than 1.3.31 is available: 1.4.31</td>
    </tr>
  </tbody>
</table>